hutch_python/epics_arch.py

```diff
@@ -7,40 +7,62 @@
 import happi
 from happi.backends.qs_db import QSBackend
 
-import hutch_python.constants
-
 from .constants import EPICS_ARCH_FILE_PATH
 
 logger = logging.getLogger(__name__)
 
-# Simple Argument Parser Setup
-parser = argparse.ArgumentParser(description='Create an epicsArch file from'
-                                             ' the Quesionnaire')
 
-parser.add_argument('experiment', nargs='?', help='Experiment name to'
-                    ' create the epicsArch file from. E.g.: xpplv6818')
+def _create_parser():
+    """Argument Parser Setup. Define shell commands."""
+    parser = argparse.ArgumentParser(description='Create an epicsArch file '
+                                                 'from the Quesionnaire')
+
+    parser.add_argument('experiment', help='Experiment name to'
+                        ' create the epicsArch file from. E.g.: xpplv6818')
+
+    parser.add_argument('--hutch', action="store",
```

Comment:
So should I have some proper `default` values here for the `--hutch` and `--path`? I am not sure what they should be

Reply:
They should reflect the defaults you have in `create_arch_file` - `None` is correct here.

As far as the help string - the key to this parameter is that it overrides detection from the experiment name, isn't it?

Reply:
yes that's correct, need to update that. 
Here if the experiment is `xpplv6818` and `--hutch xcs` will write to `/cds/group/pcds/dist/pds/xcs/misc/` and the file will be called `ecpisArch_xpplv6818`

Reply:
Updated this in af104ac